charts/hono/templates/hono-service-device-registry-base/hono-service-device-registry-post-install-job.yaml

```diff
@@ -48,6 +48,6 @@ spec:
       volumes:
       - name: post-install-data
         configMap:
-          name: {{ printf "%s-post-install-device-registry-conf" .Release.Name }}
+          name: {{ include "hono.fullname" . }}"-post-install-device-registry-conf"
```

Comment:
this produces `"hono\"-post-install-device-registry-conf\"` which causes the deployment test to fail

I suggest to use
```yaml
name: {{ printf "%s-post-install-device-registry-conf" ( include "hono.fullname" . ) }}
```

Reply:
fixed it for al occurrences of the same thing